src/pages/_app.js

```diff
@@ -83,10 +83,17 @@ const ApplicationLayout = ({ children }) => {
       const url = new URL(
         `${window.location.protocol}//${window.location.host}${path}`,
       );
+      const query = Object.fromEntries(url.searchParams.entries());
       const hasPage = url.searchParams.has('page');
-      router.push({ pathname: url.pathname, query: {} }, path, {
-        shallow: hasPage,
-      });
+      if (hasPage) {
+        window.history.pushState(
+          undefined,
+          '',
+          `${window.location.protocol}//${window.location.host}${path}`,
+        );
```

Comment:
Can you create a follow-up ticket to look into the following scenario?

- Click search in menu
- Go to page 2 of search results
- Click search in menu again => Nothing happens

I know the context from the call we did and we don't need to fix it in this PR, but it should be investigated if/how we can solve that.

Reply:
It will also be an issue for other pages, like:

- Click roles in the menu
- Go to page 2 of roles
- Click roles in the menu again => Nothing happens

Reply:
Isn't this just related to [this issue](https://jira.uitdatabank.be/browse/III-3727). Because the second click it works for me

Reply:
Either way.

Here is a ticket to go over all possible routing scenario's, to then find the best possible solution: [III-3743](https://jira.uitdatabank.be/browse/III-3743)

Reply:
Might be, let's get that fixed first then!